shell/platform/common/text_editing_delta.h

```diff
@@ -0,0 +1,72 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_COMMON_TEXT_EDITING_DELTA_H_
+#define FLUTTER_SHELL_PLATFORM_COMMON_TEXT_EDITING_DELTA_H_
+#define _SILENCE_CXX17_CODECVT_HEADER_DEPRECATION_WARNING
+
+#include <codecvt>
+#include <string>
+
+#include "flutter/shell/platform/common/text_range.h"
+
+namespace flutter {
+
+struct TextEditingDelta {
```

Comment:
Add doc comments to this struct and the methods.

Reply:
Let me know if I've missed any that usually should have doc comments.  I skipped some things that seemed obvious.